### DISCUSSION TITLE
server showing previous sites background color

### DISCUSSION THREAD

**Discussion by TaylorHardaway** [nss-evening-cohort-14/evening-client-side#20] [Help Me!] [answered]
Taylor Hardaway

The other related issues with this are not quite the same.

I am pressing hs -o to pull up the server for the star wars site, but it is showing one of my 30 repos test sites. Something similar actually happened when I started my 30 repo's for the day, it was all white, however refreshing seem to help. I'm realizing now that it is still stuck on I believe that first test site of this morning. Refreshing does not seem to work.

No error messages, just pulls up wrong site

These are answers I found online, but they don't seem to answer my question.
https://stackoverflow.com/questions/23341612/why-is-git-trying-to-connect-to-the-wrong-host
https://about.gitlab.com/blog/2018/08/08/git-happens/
https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000095004-Git-Pull-pulls-from-the-wrong-repository-

I've just tried refereshing, Xing out, cltr C to stop, and try again.

https://github.com/TaylorHardaway/week1-starwarsreal
(This is the whole repo, but I'm not sure where the problem is)

**Comment by TaylorHardaway** [comment, marked as answer]
Ok, so it looks like a "hard refresh" was the problem. On Windows it's ctrl + f5.

**Reply by drteresavasquez** [reply]
That is what I was going to suggest ;-) Great figuring it out!
